src/drivers/bitbucket_cloud.js

```diff
@@ -22,43 +22,67 @@ class BitBucketCloud {
 
   async commentCreate(opts = {}) {
     const { projectPath } = this;
-    const { commitSha, report } = opts;
-
-    // Make a comment in the commit
-    const commitEndpoint = `/repositories/${projectPath}/commit/${commitSha}/comments/`;
-    const commitBody = JSON.stringify({ content: { raw: report } });
-    const commitOutput = await this.request({
-      endpoint: commitEndpoint,
-      method: 'POST',
-      body: commitBody
-    });
+    const { commitSha, report, update, watermark } = opts;
 
     // Check for a corresponding PR. If it exists, also put the comment there.
-    const getPrEndpt = `/repositories/${projectPath}/commit/${commitSha}/pullrequests`;
-    const { values: prs } = await this.request({ endpoint: getPrEndpt });
+    let prs;
+    try {
+      const getPrEndpoint = `/repositories/${projectPath}/commit/${commitSha}/pullrequests`;
+      prs = await this.paginatedRequest({ endpoint: getPrEndpoint });
+    } catch (err) {
+      if (err.message === 'Not Found Resource not found')
+        err.message =
+          "Click 'Go to pull request' on any commit details page to enable this API";
```

Comment:
maybe also add a link to the relevant BB docs which state this (for inquisitive user reference)?

Reply:
@DavidGOrtega seems to dislike errors with links, but I actually find them an awesome resource on modern terminal emulators.

Even if it's out of scope for this pull request, I would love to see links like `https://err[or].cml.dev/YpQ` on those errors that are especially tricky to explain inline or require additional context. If I recall correctly, DVC does something similar.

Reply:
error message with link prevents a user opening an issue "I got this error saying I haz to click somewhere can u plz automate it"

Reply:
Moved to https://github.com/iterative/cml/issues/606